app/controllers/ticket_controller.rb

```diff
@@ -54,13 +54,13 @@ def make_reservation
     end
 
     def confirm_reservation
-      Ticket.where(:paid => false).update_all(paid: true)
+      Ticket.where(paid: false).update_all(paid: true)
```

Comment:
You probably want to scope it to just a single ticket, not all unpaid tickets.

Reply:
In your case you will pay for your family tickets and everybody else that are going to the cinema but not yet paid ;)

Reply:
@jandudulski Ok, now i see that. How about if i add ecrypted passcode atribute to database?  and not show its column in index?